partEZ/app/Http/Controllers/EventController.php

```diff
@@ -456,6 +474,17 @@ public static function sendInvitation($eid, $email, $uid)
         });
     }
 
+    public static function sendCancellation($eid, $email)
+    {
```

Comment:
Just a thought, but we should have an email controller that does all the email stuffs